okio/src/commonMain/kotlin/okio/internal/AbstractMessageDigest.kt

```diff
@@ -28,6 +28,7 @@ internal abstract class AbstractMessageDigest : OkioMessageDigest {
     byteCount: Int
   ) {
     val bytes = unprocessed + input.toBytes().slice(offset until offset + byteCount)
+    unprocessed = Bytes.EMPTY
```

Comment:
FYI @kevincianfarini I found a bug where we weren’t clearing unprocessed if the unprocessed and new data added up to an exact multiple of 64 !

Reply:
Ouch! It had actually occured to me earlier today that my PR was lacking tests that called update multiple times. 

Great catch!